data/net/minecraft/world/item/alchemy/PotionUtils.mapping

```diff
@@ -1,38 +1,38 @@
 CLASS net/minecraft/world/item/alchemy/PotionUtils
 	METHOD addPotionTooltip (Lnet/minecraft/world/item/ItemStack;Ljava/util/List;F)V
-		ARG 0 item
-		ARG 1 lores
+		ARG 0 stack
+		ARG 1 tooltips
 		ARG 2 durationFactor
 	METHOD getAllEffects (Lnet/minecraft/nbt/CompoundTag;)Ljava/util/List;
 		COMMENT Creates a list of PotionEffect from data on a NBTTagCompound.
-		ARG 0 tag
+		ARG 0 compoundTag
```

Comment:
More of a meta comment: I'd argue the replacing of `tag` with `compoundTag` is unnecessary in most cases here. I would certainly not advocate for either one as a required standard but wanted to see if anyone else had strong opinions here.

Reply:
Closing this comment, we'll have to discuss this during a meeting.
Put a note of this question in the standards discussion channel.